artmaker/src/main/java/io/artmaker/ArtMaker.kt

```diff
@@ -78,18 +84,40 @@ fun ArtMaker(modifier: Modifier = Modifier, onFinishDrawing: (Bitmap) -> Unit =
         floatingActionButton = {
             AnimatedVisibility(
                 visible = viewModel.pathList.isNotEmpty(),
-                modifier = Modifier.padding(bottom = dimensionResource(id = R.dimen.Padding60)),
+                modifier = Modifier.padding(
+                    bottom = if (isFullScreenEnabled) {
+                        dimensionResource(id = R.dimen.Padding0)
+                    } else {
+                        dimensionResource(id = R.dimen.Padding60)
+                    },
+                ),
             ) {
                 Column {
                     // Trigger sharing the image. It has to save the image first
-                    FloatingActionButton(onClick = { viewModel.onAction(ArtMakerAction.TriggerArtExport(ExportType.ShareImage)) }) {
-                        Icon(imageVector = Icons.Filled.Share, contentDescription = Icons.Filled.Share.name)
+                    if (artMakerConfiguration.canShareArt) {
+                        FloatingActionButton(onClick = { viewModel.onAction(ArtMakerAction.TriggerArtExport(ExportType.ShareImage)) }) {
+                            Icon(imageVector = Icons.Filled.Share, contentDescription = Icons.Filled.Share.name)
+                        }
                     }
                     Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.Padding4)))
                     // Finish the drawing and hand it back to the calling application as a bitmap
                     FloatingActionButton(onClick = { viewModel.onAction(ArtMakerAction.TriggerArtExport(ExportType.FinishDrawingImage)) }) {
                         Icon(imageVector = Icons.Filled.Done, contentDescription = Icons.Filled.Done.name)
                     }
+                    Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.Padding4)))
+                    FloatingActionButton(onClick = { isFullScreenEnabled = !isFullScreenEnabled }) {
+                        if (isFullScreenEnabled) {
```

Comment:
You can simplify this as follows:

`                    FloatingActionButton(onClick = { isFullScreenEnabled = !isFullScreenEnabled }) {
                        Icon(
                            imageVector = if (isFullScreenEnabled) Icons.Filled.Fullscreen else Icons.Filled.FullscreenExit,
                            contentDescription = if (isFullScreenEnabled) Icons.Filled.Fullscreen.name else Icons.Filled.FullscreenExit.name,
                        )
                    }`